### ISSUE TITLE
tf.data.Dataset.from_genertor(generator_fn,output_shapes,output_types,args(sents1,sents2))

### ISSUE THREAD

**Issue by DrBugKiller** [Kyubyong/transformer#73]
Here is a little question.The class of sents1 and sents2 is list of str,but in generator_fn,the class of sents1 and sents2 is list of bytes,why?Is that the effect of  tf.data.Dataset.from_genertor?

**Comment by ty5491003** [comment]
Yes, i met some question. Then i readed the doc of tensorflow.data.Dataset.from_generator(), but i didn't finded the reason about 'byte'. But when i search 'byte' on the page, i finded: 
> For large datasets (> 1 GB), this can waste memory and run into byte limits of graph serialization.
So i guess from_generator() has same  mechanism.
